manifests/config_cmd.pp

```diff
@@ -30,21 +34,22 @@
     $base_cmd = "qpid-config -b amqp://${hostname}:${_port}"
   }
 
+  Service <| tag == 'qpidd' |> ->
```

Comment:
If I understand correctly, this will make it usable where the qpid-client and qpid server are on separate hosts?